src/freenas/usr/bin/unlock-root-fs.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+
+import json
+import sys
+from subprocess import run
+
+
+def unlock_dataset(entry):
```

Comment:
Lock/unlock are the terms used for encryption. Can we come up with another term for toggling readonly flag on the filesystem?

Reply:
Sure.

Reply:
I renamed the script as `truenas-developer-enable.py`